src/core/Akka.TestKit/TestKitBase.cs

```diff
@@ -173,6 +173,15 @@ protected void InitializeTest(ActorSystem system, Config config, string actorSys
         /// </summary>
         public EventFilterFactory EventFilter { get { return _testState.EventFilterFactory; } }
 
+        /// <summary>
+        /// Creates a new event filter for the specified actor system.
+        /// </summary>
+        /// <param name="system">Actor system.</param>
+        /// <returns>A new instance of <see cref="EventFilterFactory"/>.</returns>
+        public EventFilterFactory CreateEventFilter(ActorSystem system)
```

Comment:
Looks good - so this means that the default `EventFilter` will still work exactly as expected then. That was the only major thing I wanted to check first.